### DISCUSSION TITLE
Labs: Hero 11 Mini stops recording all the time...would donate you if someone can help me quickly - thank you

### DISCUSSION THREAD

**Discussion by maxcineflights** [gopro/labs#227] [Q&A]
Hello guys I have a Problem:
Hero 11 Mini:

    5.3k 30FPS
    10bit colors
    Max/MIN ISO 100
    WB: 5500K
    Sharpness: LOW

Labs settings:

    NR01=1
    BITR=180

SanDisk Extreme Pro, 128GB, exFAT

yesterday i tried 3 times with 8 bit (everything worked till i stopped the recording after around 5 minutes, worked flawlessly...

today I tried it with 10 bit on, any my recordings 5/10 have stopped randomly sometimes it worked for 5-6 minutes sometimes after 3 minutes it broke up....

what can i change to still have 10 bit and NR=1 (which Bitrate is reliable?) or is it maybe because of the 10 bit or because of the NR=1? (I think NR=1 = off should make less Processing power for the GoPro or am I wrong?) 

The sad part is, I though everything works perfectly and I travel south east asia right now, so there is no Time for flying in the same spot a few times....

I really want a field testet setting that never stops my recording (FPV Flying, turn the GoPro on and start recording when everyhting else is ready for takeoff..)

Thank you guys...

**Comment by HiStephenT** [comment]
Looks like your SD card can’t keep up, try BITR=150
Out of curiosity test your sd card speed using SPED=3 this will test the write speed 3 times it will take 30secs to complete. SPED=1 will take 10secs

**Comment by maxcineflights** [comment]
I tried the speed test a few times before already, (but end results are gone pretty fast on mini11 it was always around 480-500 use SanDisk Extreme Pro.

If I go for example with BITR=150 is it still possible to record in 5k 30, 8:7 + 10BIT + NR=1 ?? (think the NR=1 needs more datarate but not sure at what threshold it will be making problems and how these problems woud affect my footage if my bitrate is getting to low for NR=1... the weird part is, that it sometimes works...

Is there any way to read out kindoff a "failcode" of the last fail to see whats the problem?

**Comment by maxcineflights** [comment]
the card is even reccomended from GoPro itself I think so...
is there maybe any source about what cards work reliably with 180 bitrate? (but I even think if it works with 150bitrate i should be good, 30 bit will not be that visible or am i wrong?
Normally I love the SanDisk Extreme Pros because I never had any problems and they are all brand new...have made bad experience already with Samsung Cards for example.

**Reply by dnewman-gpsw** [reply]
The camera is only rated to 120Mb/s, Labs is simply pushing those boundaries into untested areas.  SanDisk Extreme Pro is plenty fast enough, is more a factor that the camera is extremely busy, and can't use all the speed.  In the SPEeD test, the camera is not encoding, so more bandwidth for writing is available.  The Speed test was more useful indicator for GP1 cameras, which had much more limited I/O bandwidth.  

NR01=1 doesn't bypass noise reduction, it just turns it down to be effectively off. So this is not a thermal saving, but it wouldn't make the camera's work easier, as these denoising blocks are in hardware, they are part of the processing pipeline, bypassing them has more risk. However, having more noise makes the encoder bit-rate more volatile, so moment with a lot of sky switching to a lot of texture (grass, leaves, etc.) will make the bitrate temporarily climb (same for ISO changes), remember if 180Mb/s is your target average, the data rate will vary +/- 20Mb/s throughout the capture.  

As for compression quality, there are visible advantages up to 150Mb/s, but then there are diminishing returns. It is nearly impossible to watch 150Mb/s and 180Mb/s video side-by-side and see the difference. I understand you want the best image possible, but as your shooting environment and camera if different then everyone else, it is up to you to determine the bit-rate that works reliably.  

This is the warning message for Bones users (with no 10-bit or 8:7, both increasing the workload)
![image](https://user-images.githubusercontent.com/10856084/216838039-d676ce98-e1db-4ccf-9bcb-011735c1acf6.png)

You could try disabling audio and/or LRVs.  Their data rates are tiny, but I would be curious to hear if it helped.

**Reply by dnewman-gpsw** [reply]
(from the other thread) Unfortunately the qrlog.txt will not report a capture failure due to write speed, as it is not within Labs code. There is no error message returned from a failed command, as the command to set a new bitrate target worked. The camera doesn't have a bit-rate too high message (like battery too low, or temp too high) as it is design to run within safe limits (120Mb/s max.)  Labs BITR control is like taking the speed-limiter off on a car, it might go faster, but all the safety equipment isn't designed for that higher speed.

**Reply by maxcineflights** [reply]
I have testet the internal labs speedtest multiple times with different cards (all SanDisk Extreme Pro) its always around 580-620 mbs so fast enough. Its 100% no overheat issue. Today I have done 7 flights, first 6 no problems with 160BITR and NR=1 but the last flight stopped again after 1 minute (I was anyway flying back, because the shot was too risky and just saw randomly the first time the message : "SD CARD TOO SLOW USE U3 SD CARD" (or something really similar like that) the funny part is that it is a u3 and its even recommended from GoPro itself...I don't know, I will now try 150 but it's really frustrating loosing recordings all the time...IT IS NO OVERHEAT AND NO FREEZE...its so weird. 


"As for compression quality, there are visible advantages up to 150Mb/s, but then there are diminishing returns. It is nearly impossible to watch 150Mb/s and 180Mb/s video side-by-side and see the difference. I understand you want the best image possible, but as your shooting environment and camera if different then everyone else, it is up to you to determine the bit-rate that works reliably."

Thank you so much! I try now 150 and see if its reliable. (its just so annoying because 180 is the bitrate that I can see everyone is using stable, at least on YT, FB and here on GitHub too) 



"This is the warning message for Bones users (with no 10-bit or 8:7, both increasing the workload)" - That would also make sense to add as Information to H11/11 mini, because this would have made it in the beginning more clearly to me, even if its logical, it was so confusiong because I was in the beinning not sure if its, my gopro, my sd or the labs fw or whatever...


You could try disabling audio and/or LRVs. Their data rates are tiny, but I would be curious to hear if it helped.
the Problem is I want my Audio in my Recordings, even if i probably never use it, I say my cam settings sometimes in the beginning of a recroding to myself just to keep track what i have done different whit different lighting condigtions etc. 

+ its now with 160 definitly more stable as with 180...
will now try 150


Thank you so much for your help and Input was pretty interestting and helpful to me!
